### PR TITLE
Team Value PRs

### PR DESCRIPTION
**Pull request by sandeepmehta2155**

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -1,4 +1,249 @@
...
+  left: 2rem;
+  display: block;
+  width: 15%;
+  max-height: 80%;
+  overflow-y: none;
+  overflow-x: scroll;
+  text-align: left;
+}
+.navigationPills {
+  /* margin: 5rem; */
```
Comment (rohan9896) [inline src/styles.css]: remove this comment

**Review by rohan9896 — CHANGES_REQUESTED** [review]
Hey sandeep! I checked your code and I have to say that code is very clean and also you have re-used your components alot, that's why the lines of code is very less which is great

fixes - 
- side bar is not responsive
- You haven't added redirects file bcoz of which it is going to page not found on reload
- Pause button not working 

Some features you can add later - 
- Search functionality
-  separate 404 page
- Add a logo (maybe)

**Review by kajol-rep — APPROVED** [review]
Hi Sandeep,
reviewed your code, it looks clean.
I would like to suggest few things that will help you to make your UI better:
1. You can fix margin for your sidebar so that subscribers doesn't go to next line.
2. Make sure to keep UI clean, for this you can hide scrollbars if its not scrollable.
3. Make sure whenever the user clicks on thumbnail image or content it should go to video player page and it should be implemented on all the pages. As pages except home doesn't have this feature.

**Comment by sandeepmehta2155** [comment]
> Hey sandeep! I checked your code and I have to say that code is very clean and also you have re-used your components alot, that's why the lines of code is very less which is great
> 
> fixes -
> 
> * side bar is not responsive
> * You haven't added redirects file bcoz of which it is going to page not found on reload
> * Pause button not working
> 
> Some features you can add later -
> 
> * Search functionality
> * separate 404 page
> * Add a logo (maybe)

All issues have been resolved, can review it further for whatever is build

**Comment by sandeepmehta2155** [comment]
> Hi Sandeep,
> reviewed your code, it looks clean.
> I would like to suggest few things that will help you to make your UI better:
> 
> 1. You can fix margin for your sidebar so that subscribers doesn't go to next line.
> 2. Make sure to keep UI clean, for this you can hide scrollbars if its not scrollable.
> 3. Make sure whenever the user clicks on thumbnail image or content it should go to video player page and it should be implemented on all the pages. As pages except home doesn't have this feature.

**Comment by sandeepmehta2155** [comment]
> Hi Sandeep,
> reviewed your code, it looks clean.
> I would like to suggest few things that will help you to make your UI better:
> 
> 1. You can fix margin for your sidebar so that subscribers doesn't go to next line.
> 2. Make sure to keep UI clean, for this you can hide scrollbars if its not scrollable.
> 3. Make sure whenever the user clicks on thumbnail image or content it should go to video player page and it should be implemented on all the pages. As pages except home doesn't have this feature.

Thanks Kajol for enhancing small pieces of UI resulting into a clean &  impressive app ,further acknowledge me for changes made or any changes required to be made.

**Comment by kajol-rep** [comment]
> > Hi Sandeep,
> > reviewed your code, it looks clean.
> > I would like to suggest few things that will help you to make your UI better:
> > 
> > 1. You can fix margin for your sidebar so that subscribers doesn't go to next line.
> > 2. Make sure to keep UI clean, for this you can hide scrollbars if its not scrollable.
> > 3. Make sure whenever the user clicks on thumbnail image or content it should go to video player page and it should be implemented on all the pages. As pages except home doesn't have this feature.
> 
> Thanks Kajol for enhancing small pieces of UI resulting into a clean & impressive app ,further acknowledge me for changes made or any changes required to be made.

Thanks for considering my suggestions. Surely will acknowledge you further for any changes.

**Review by rohan9896 — APPROVED** [review]
Well done sandeep😃

**Pull request merged**